modules/oe_content_event/src/EventUtilities.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_event;
+
+/**
+ * Provides generic event helper methods.
+ */
+class EventUtilities {
```

Comment:
Since this is generic, and not bound to the event content type only, please make it a static class, rename it as below  and move this to the following location:

```php
Drupal\oe_content_event\Utilities

/**
 * Provides helper methods to manipulate content forms.
 */
class ContentFormUtilities {
```